course3/homework-vector/vector.cc

```diff
@@ -3,9 +3,59 @@
 
 class Vector
 {
-    // TODO
```

Comment:
please, sent assignemtnsd in separate pul requests. keep them in separate branches